publish/tests/test_version.py

```diff
@@ -30,6 +31,25 @@ def test_version_from_girder(dandiset_factory, mock_girder_client):
     assert version
 
 
+@pytest.mark.django_db
+def test_version_from_girder_no_metadata(dandiset_factory, mock_girder_client):
+    dandiset = dandiset_factory()
+    with pytest.raises(ValidationError) as excinfo:
+        Version.from_girder(dandiset, mock_girder_client)
+    assert (
```

Comment:
Just use the [`match` keyword argument to `pytest.raises`](https://docs.pytest.org/en/stable/reference.html#pytest-raises), and you shouldn't need a separate assert. Also, feel free to only match a substring of the exception (if it makes it more convenient), as long as it gives you confidence that it's the exact exception you're expecting.

Reply:
Resolved in https://github.com/dandi/dandi-publish/pull/63/commits/78c889e0fcad96bc9740c26fc69c3f6ed240e7ce